crates/nu-command/src/platform/ansi/ansi_.rs

```diff
@@ -382,7 +382,7 @@ Format: #
         } else if osc && param_is_valid_string {
             // Operating system command aka osc  ESC ] <- note the right brace, not left brace for osc
             // OCS's need to end with a bell '\x07' char
```

Comment:
Is this comment true in all cases, e.g. the hyperlink case? It looks like `ansi st` is something else

```
~ 〉ansi st | hexyl
┌────────┬─────────────────────────┬─────────────────────────┬────────┬────────┐
│00000000│ 1b 5c                   ┊                         │•\      ┊        │
└────────┴─────────────────────────┴─────────────────────────┴────────┴────────┘
```

Reply:
No, I've learned that since I wrote that comment that either `bel` or `st` are usually acceptible.

Reply:
Ah OK interesting. It's still a bit of an alien world to me even though I had to learn a bit for delta. (Would you mind updating this comment?)